beekeeper-path-scheduler-apiary/pom.xml

```diff
@@ -62,6 +62,18 @@
           <groupId>org.apache.logging.log4j</groupId>
           <artifactId>log4j-slf4j-impl</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>javax.servlet</groupId>
```

Comment:
Removed `servlet-api` and `javax.servlet` as the ITs wouldnt start (though the applications would).

Removed `geronimo-jaspic_1.0_spec` because calling the endpoint `/actuator/health` during the ITs threw this error (though was fine when running the applications):

```
java.lang.SecurityException: AuthConfigFactory error: java.lang.ClassNotFoundException: org.apache.geronimo.components.jaspi.AuthConfigFactoryImpl
```

did some Googling, didn't find much. But removing this package meant the tomcat equivalent `AuthConfigFactoryImpl` was used, and that runs fine.